### DISCUSSION TITLE
Slide does not opens

### DISCUSSION THREAD

**Discussion by adarsh7raj** [camicroscope/caMicroscope#688] [Q&A]
I uploaded these slides (given below) but when i try to open it just hangs there and the slide does not opens

https://github.com/camicroscope/caMicroscope/assets/130682848/5d327866-03bd-4306-813e-f4e2503d4367

i uploaded the file which is in svs extension and of size 265 bytes and in some of the documentation i read that to upload a slide you must have a WSI file i think i do not have WSI file how to get that?

**Comment by birm** [comment]
Can you tell me about the file you added? 265 bytes sounds really small!
To get Slides, openslide has some examples that I tend to use for basic testing (https://openslide.cs.cmu.edu/download/openslide-testdata/Aperio/)

**Reply by gbengaoluwadahunsi** [reply]
thank they Birm

**Comment by sarahannie** [comment]
@birm 
![Screenshot (192)](https://github.com/camicroscope/caMicroscope/assets/87903267/4b0f9856-de1e-41ba-81fe-c5514c1f54ea)
i keep getting this message when i try to upload slide using your reference link above

**Reply by birm** [reply]
a .crdownload file is a file that chrome makes as a placeholder during download; you should wait for it to finish downloading

**Comment by sarahannie** [comment]
follow up question when i tried input the filename and all the necessary step i got status code error 500 what the way out or is their error coming from my installation or process method step error?? 
![Screenshot (193)](https://github.com/camicroscope/caMicroscope/assets/87903267/b6bb0315-0c3e-426d-9a67-a16b37df9d1a)

**Reply by birm** [reply]
I think your load service crashed or never built. Can you see if there is anything helpful in its logs (`docker logs ca-load`)
